StressTest/GetTest.yml

```diff
@@ -0,0 +1,15 @@
+config:
+  target: 'http://localhost:80'
+  http:
+    pool: 10
+  phases:
+    - duration: 60
+      arrivalRate: 6000
+  defaults:
+    headers:
+      x-my-service-auth: '987401838271002188298567'
```

Comment:
I think this is from the documentation, I'm assuming you don't need this as a header